corehq/ex-submodules/dimagi/utils/logging.py

```diff
@@ -7,7 +7,6 @@
 
 
 def notify_error(message, details=None):
-    print({'details': details})
```

Comment:
I'm assuming this was not meant to stay in the code: https://github.com/dimagi/commcare-hq/commit/f7deb18458a65ccc2ab7671f32c3de2870987c26#diff-d8e4819f0f2d300a1cb8fe516e5ca075d82d85d9f0666cf634d36c9644be17bfR10